src/config.py

```diff
@@ -78,3 +78,35 @@ def get_image_fullname(image_name, version, image_conf, env_conf):
 
     return f"{image_repo_name_base}:{image_tag}"
 
+
+def get_image_tags(image_name, version, image_conf, env_conf):
+    image_repo_name_base = f"{image_conf['docker_hub_namespace']}/ci-{image_name}"
+    local_repo_name_base = f"localhost:5000/ci-{image_name}"
+    version_tag = f'{version}-' if version != "1" else ""
+
+    if env_conf["tag"]:
+        version_tag += env_conf["tag"] if version != "1" else env_conf["tag"]
+    elif env_conf["event_type"] == "schedule":
+        version_tag += "nightly" if version != "1" else "nightly"
+    elif env_conf["branch"] in ["master"]:
+        version_tag += "latest" if version != "1" else "latest"
+    else:
+        version_tag += "latest" if version != "1" else "latest"
```

Comment:
Is this not twice the same thing? I'm reading it as
```python
version_tag += ("latest" if version != "1" else "latest")
```
so it's `"latest"` or `"latest"`? Or is my Python so rusty?

Reply:
I asked myself the same question, I took over the construction algo of the master branch. I will do other tests to see if we can simplify in the following versions.

Reply:
I've found [the refactoring](https://github.com/ekino/docker-buildbox/commit/642e8bee4b3b7f560eedd07bbfc30ee1573ab134#diff-24a3fe5c783755412cb9de83238b4dfd0699345bf50dcadc3ec23c6749361592) which introduced this oddity. It used to be `"-latest" else "latest"`, but the hyphen was moved to the prefix.

So all these constructs can be simplified to `"latest"` (or `"nightly"` or `env_conf["tag"]` depending on the line), both in `get_image_fullname` and `get_image_tags`.

Reply:
Done !
Commit id : faef7d25b4a3f47cad021c08b4ad614456d423f9